packages/view/backend/sirius-components-view-emf/src/main/java/org/eclipse/sirius/components/view/emf/diagram/DiagramOperationInterpreter.java

```diff
@@ -46,21 +48,28 @@ public class DiagramOperationInterpreter implements IOperationInterpreter {
 
     private final IDiagramContext diagramContext;
 
+    private final IFeedbackMessageService feedbackMessageService;
+
     private final Map<org.eclipse.sirius.components.view.NodeDescription, NodeDescription> convertedNodes;
 
     public DiagramOperationInterpreter(AQLInterpreter interpreter, IObjectService objectService, IEditService editService, IDiagramContext diagramContext,
-            Map<org.eclipse.sirius.components.view.NodeDescription, NodeDescription> convertedNodes) {
+            Map<org.eclipse.sirius.components.view.NodeDescription, NodeDescription> convertedNodes, IFeedbackMessageService feedbackMessageService) {
         this.interpreter = Objects.requireNonNull(interpreter);
         this.objectService = Objects.requireNonNull(objectService);
         this.editService = Objects.requireNonNull(editService);
         this.diagramContext = diagramContext;
         this.convertedNodes = Objects.requireNonNull(convertedNodes);
+        this.feedbackMessageService = feedbackMessageService;
     }
 
     public IStatus executeTool(Tool tool, VariableManager variableManager) {
         Optional<VariableManager> optionalVariableManager = this.executeOperations(tool.getBody(), variableManager);
         if (optionalVariableManager.isEmpty()) {
-            return new Failure(String.format("Something went wrong while executing the tool '%s'", tool.getName()));
+            var errorMessages = new ArrayList<>(List.of(String.format("Something went wrong while executing the tool '%s'", tool.getName())));
```

Comment:
`errorMessages` => `feedbackMessages`

Reply:
Does this mean that if the tool sends an "INFO"-level message it will get prefixed by "Something went wrong while executing the tool"?
If yes, this seems wrong.